target/linux/mvebu/base-files/etc/uci-defaults/04_mambafan

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+#
+# Copyright (C) 2017 LEDE-Project.org
+#
+ 
+. /lib/mvebu.sh
+ 
+board=$(mvebu_board_name)
+ 
+case "$board" in
+armada-xp-linksys-mamba)
+
+	cat > /etc/init.d/fan_control << 'FC_EOF'
+#!/bin/sh /etc/rc.common
+
+# mambafan daemon control script (/etc/init.d/fan_control)
+# for start/stop of mamba fan monitor (/usr/sbin/fan_monitor)
+
+START=99
+STOP=99
+
+PIDFILE=/var/run/fan_monitor_pid
+
+start() {
+	start-stop-daemon -b -S -x /usr/sbin/fan_monitor -m -p $PIDFILE
+}
+
+stop() {
+	start-stop-daemon -K -p $PIDFILE
+}
+FC_EOF
+
+	cat > /usr/sbin/fan_monitor << 'FM_EOF'
```

Comment:
Same comment as to your previous PR: I do not like the approach where you create this script /usr/sbin/fan_monitor on the fly in uci-defaults, causing it to be placed on flash twice after the boot.

You could have this /usr/sbin/fan_monitor script always in /usr/sbin/
It would just be useless there in non-mamba devices, but it would cause no harm. That would also remvoe one permission setting command from the process.

I understand the uci-defaults approach concerning the init script /etc/init.d/fan_control that should only be created in mamba.